zip_example/TRMNL.sh

```diff
@@ -58,6 +59,14 @@ eips_debug() {
   fi
 }
 
+init() {
+  /etc/init.d/framework stop
+  initctl stop webreader >/dev/null 2>&1
+  echo powersave >/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
+  lipc-set-prop com.lab126.powerd preventScreenSaver 1
```

Comment:
I'm pretty sure frontlight problem comes from this line, I have to check if it impacts the refresh if I remove it.